test/sql/export/export_quoted_structs.test

```diff
@@ -11,8 +11,22 @@ BEGIN TRANSACTION
 statement ok
 create table a (s STRUCT("end" VARCHAR));
 
-statement ok
+statement error
```

Comment:
I'm not sure if this keeps the original behavior of the test?

Reply:
I reset the test after this so it stays the same but it's probably unnecessary to trigger an error here.